docker-hub/builds/index.md

```diff
@@ -277,6 +277,11 @@ You could also use capture groups to build and label images that come from vario
 
 `/(alice|bob)-v([0-9.]+)/` -->
 
+### Build image with Buildkit
+
+You can enable the Buildkit builder by setting the environment variable `DOCKER_BUILDKIT=1`([enviroment variables](index.md#environment-variables-for-builds)) in the Configure automated build section ([configure builds](index.md#configure-automated-build-settings)).
+For more information on Buildkit please see [buildkit](https://docs.docker.com/develop/develop-images/build_enhancements/).
```

Comment:
Try to avoid "please"; also, we should avoid using a hard-coded URL for docs.docker.com (so that preview builds of the docs link correctly), perhaps;

```suggestion
Refer to the [build images with BuildKit](../../develop/develop-images/build_enhancements.md)
page for more information on BuildKit.
```